kubeutil.py

```diff
@@ -111,6 +111,10 @@ def create_extensions_v1beta1_api(self):
         cfg = self.get_kubecfg()
         return client.ExtensionsV1beta1Api(cfg)
 
+    def get_version_info(self):
```

Comment:
rename: (get_?)cluster_version_info 
mv out of Ctx class. either introduce a new helper class, or make this a module function